$NetBSD: patch-config.h.meson,v 1.1 2021/02/25 09:55:55 ryoon Exp $

* meson's has_function misdetects sincos(3) under NetBSD 9.99.80 at least.
  Do not define HAVE_SINCOS for NetBSD in ugly way.

--- config.h.meson.orig	2021-02-07 23:06:57.305976200 +0000
+++ config.h.meson
@@ -86,7 +86,9 @@
 #mesondefine HAVE_RANDR15
 
 /* Define to 1 if you have the `sincos' function. */
+#if !defined(__NetBSD__)
 #mesondefine HAVE_SINCOS
+#endif
 
 /* Define to 1 if you have the <stdint.h> header file. */
 #mesondefine HAVE_STDINT_H
